src/core/core.controller.js

```diff
@@ -769,8 +773,17 @@ helpers.extend(Chart.prototype, /** @lends Chart */ {
 			return;
 		}
 
+		helpers.canvas.clipArea(ctx, {
+			left: clip.left === false ? 0 : area.left - clip.left,
```

Comment:
I'm surprise the default is `false` instead of `null` or `undefined`

Reply:
Default is `false` only for controllers not overriding `getMaxOverflow`
`false` meaning no clipping. Because the clipping could be off on one edge only, `false` is tested and clipping area extended to that edge of `canvas`.